packages/compass-preferences-model/src/global-config.ts

```diff
@@ -260,7 +261,9 @@ export function getHelpText(): string {
     text += formatSingleOption(key, 'cli');
   }
   text +=
-    '\n(Options marked with (*) are also configurable through the global configuration file.)\n';
+    '\nOptions marked with (*) are also configurable through the global configuration file.\n';
+  text +=
+    'Boolean options can be negated by using a `--no` prefix, e.g. `--no-network-traffic`.\n';
```

Comment:
When I'm running the app locally and print help if shows switches in camelCase:

<img width="666" alt="image" src="https://user-images.githubusercontent.com/5036933/191517598-8310f198-f065-454b-9146-bbbad7dda844.png">

should this additional help match this and be `no-networkTraffic` to make it easier to connect the two?

Reply:
Ahh, now I get why we are listing them like that, thanks! Hmm, yeah, I do like the look of dashed arguments more too, maybe it's worth making the \* description more verbose so that we can have them listed in kebab case 😅